autobahn/websocket/protocol.py

```diff
@@ -657,11 +665,10 @@ def onCloseFrame(self, code, reasonRaw):
                 # When we are a client, the server should drop the TCP
                 # If that doesn't happen, we do. And that will set wasClean = False.
                 if self.serverConnectionDropTimeout > 0:
-                    call = txaio.call_later(
+                    self.serverConnectionDropTimeoutCall = txaio.call_later(
```

Comment:
Ah;) Maybe we should make it a BatchTimer too. Also, as this will this be public API, I think it's worth to think a little about naming and such. So I'd like to propose:

* `ping_send_timer`: This timer is used for automatic, periodic sending of WebSocket pings.
* `pong_timeout_timer`: This timer is used for timeout receiving pongs to automatic pings.
* `opening_handshake_timeout_timer`: This timer is used for timout on finishing the initial WebSocket opening handshake.
* `drop_connection_timeout_timer`: This timer is used for timeout the peer automatically dropping the connection as it should happen after the WebSocket closing handshake has completed.